integrationtests/tests/fixtures/mod.rs

```diff
@@ -840,7 +846,7 @@ impl FederationTest {
 
         for server in &self.servers {
             block_on(async {
-                let svr = server.borrow_mut();
+                let svr = server.lock().await;
                 let mut dbtx = svr.database.begin_transaction().await;
```

Comment:
This loop is similar in nature to the one parallelized. Probably lower impact though, just noticed while looking at #1586.

Reply:
@elsirion Yeah, but only does IO, and database ops already yield executor so I don't think there's a reason to complicate with `spawn`.